module/modmanager/module.go

```diff
@@ -0,0 +1,424 @@
+package modmanager
```

Comment:
Did the PR meaningfully change anything in this code? Or just 100% code movement and the only functional changes are in the modmanager file?

Reply:
I did notice that `inStartup` was removed (yay!)

Reply:
The only meaningful change is removing `inStartup` and `inRecoveryLock`. The parts of the code they guarded are now handled by the manager lock.